packages/angular/projects/angular-lib/src/directives/number-value-accessor.ts

```diff
@@ -1,13 +1,13 @@
-import { Directive, ElementRef } from "@angular/core";
-import { NG_VALUE_ACCESSOR } from "@angular/forms";
+import { Directive, ElementRef } from '@angular/core';
+import { NG_VALUE_ACCESSOR } from '@angular/forms';
 
-import { ValueAccessor } from "./value-accessor";
+import { ValueAccessor } from './value-accessor';
 
 @Directive({
   /* tslint:disable-next-line:directive-selector */
-  selector: "my-input[type=number]",
+  selector: 'my-input[type=number]',
```

Comment:
@markacianfrani One of the accessor files that isn't being used, as you can see the selector is 'my-input' still. I tried a rux-input with `type=number` in an angular form (the same on on codesandbox) and it seemed to work fine, so we probably don't need this file. Leaving it here just for feedback and will delete it if necessary.